src/main/java/net/patchworkmc/patcher/util/VersionUtil.java

```diff
@@ -0,0 +1,17 @@
+package net.patchworkmc.patcher.util;
+
+public final class VersionUtil {
+	private VersionUtil() { }
+
+	public static String getMinecraftVersion() {
+		return "1.14.4";
```

Comment:
Not sure if this is the best approach, but it works for now. At some point, multiple version support would be nice.

Reply:
this is intended to allow that, in the future the global state would be mutable somehow